pkg/cache/cache.go

```diff
@@ -140,10 +143,44 @@ func (c *serviceAccountCache) ToJSON() string {
 	return string(contents)
 }
 
+func getAccountId() (string, error) {
+	sess, err := session.NewSession()
+	if err != nil {
+		klog.Errorf("Could not create new session", err.Error())
+		return "", err
+	}
+
+	metadata := ec2metadata.New(sess)
+	identity, err := metadata.GetInstanceIdentityDocument()
+	if err != nil {
+		klog.Errorf("Could not get instance identity document", err.Error())
+		return "", err
+	}
+
+	accountID := identity.AccountID
```

Comment:
No need to add a new var, I'd propose to directly return it 🙂 

```suggestion
	return identity.AccountID, nil
```